### ISSUE TITLE
unregister for an event is broken

### ISSUE THREAD

**Issue by Ana06** [openSUSE/osem#2612]
**I'm submitting a ..**
- [X] Bug Report
- [ ] Feature Request

**Current behaviour**
event.opensuse.org crashes when ana_testuser tried to unregister for openSUSE Summit Dublin.

**Comment by snpd25** [comment]
@Ana06 , I am unable to reproduce this bug in my local repository. It works fine there

**Comment by Ana06** [comment]
I don't think this has been fixed. events.opensuse.org is using ichain (that is related to other issues, so it could be the case here as well). the user I tested was is no normal user (no admin rights).

**Comment by differentreality** [comment]
Can't reproduce either :/
